tests/integration/redis-cli.tcl

```diff
@@ -109,7 +109,7 @@ start_server {tags {"cli"}} {
     test_interactive_cli "INFO response should be printed raw" {
         set lines [split [run_command $fd info] "\n"]
         foreach line $lines {
-            assert [regexp {^$|^#|^[a-z0-9_]+:.+} $line]
+            assert [regexp {^$|^#|[^#:]+:} $line]
```

Comment:
If this is supposed to match the beginning of the string up to the colon (as before) maybe this part should be `^[^#:]+:`?